resources/metadata/detect.yaml

```diff
@@ -110,6 +111,35 @@ spec:
         - PARAMETERS
         - STAGES
         - STEPS
+      - name: groups
+        description: Users groups to be assigned for the Project
+        aliases:
+          - name: detect/groups
+        type: string
+        mandatory: false
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      - name: failOn
+        description: Mark the current build as fail based the policy categories. A comma seperated list can be provided to fail on multiple categories, for eg. 'BLOCKER,CRITICAL,MAJOR'
+        aliases:
+          - name: detect/failOn
+        type: string
```

Comment:
to properly reflect the muilti-value character we should go with type `[]string`